include/flatbuffers/idl.h

```diff
@@ -406,7 +410,7 @@ struct IDLOptions {
       generate_name_strings(false),
       escape_proto_identifiers(false),
       generate_object_based_api(false),
-      cpp_object_api_pointer_type("std::unique_ptr"),
+      cpp_object_api_pointer_type("flatbuffers::unique_ptr"),
```

Comment:
I've been pondering this a bit more and perhaps I can handle stlport using macros in the generated code e.g

in stl_emulation.h do..

`
namespace std {
class flatbuffers_unique_ptr;
}
`
Then in the generated code...

`
// generated header
#ifdef FLATBUFFERS_CPP98_STL
#define unique_ptr flatbuffers_unique_ptr
#endif  // FLATBUFFERS_CPP98_STL

// generated code that uses unique_ptr
#ifdef FLATBUFFERS_CPP98_STL
#undef unique_ptr
#endif  // FLATBUFFERS_CPP98_STL
`

This is a little bit of a pain since all internal uses of std::unique_ptr will also need to be wrapped in a similar set of boilerplate which is pretty unfortunate.